src/app/[lng]/layout.tsx

```diff
@@ -103,7 +101,26 @@ export default async function RootLayout({
   children: ReactNode;
   params: { lng: string };
 }) {
-  const { resources } = await initTranslations(lng || fallbackLng, namespaces);
+  const cookiesHandler = cookies();
+  const [resourcesPromise, activeThemePromise] = await Promise.allSettled([
+    initTranslations(lng || fallbackLng, namespaces),
+    getActiveTheme(cookiesHandler),
```

Comment:
this one is to get the theme from the cookie in case the user has selection a theme before, right?

Reply:
Yes, it gets either a partner theme or another selected user theme.